I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/sceens/galery_screen.rpy", line 194, in script
    init python:
  File "game/sceens/galery_screen.rpy", line 194, in script
    init python:
  File "game/sceens/galery_screen.rpy", line 211, in <module>
    gallery.condition("ill_image_background","persistent.ill_image_3_unlocked")
TypeError: condition() takes exactly 2 arguments (3 given)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 558, in main
    renpy.game.context().run(node)
  File "game/sceens/galery_screen.rpy", line 194, in script
    init python:
  File "game/sceens/galery_screen.rpy", line 194, in script
    init python:
  File "renpy/ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/sceens/galery_screen.rpy", line 211, in <module>
    gallery.condition("ill_image_background","persistent.ill_image_3_unlocked")
TypeError: condition() takes exactly 2 arguments (3 given)

Windows-10-10.0.19041 AMD64
Ren'Py 7.5.3.22090809
Молчание коммунальных квартир 1.0
Sun Mar  5 21:39:33 2023
